CairoMakie/src/screen.jl

```diff
@@ -213,6 +214,11 @@ function apply_config!(screen::Screen, config::ScreenConfig)
     aa = to_cairo_antialias(config.antialias)
     Cairo.set_antialias(context, aa)
     set_miter_limit(context, 2.0)
+
+    if get_render_type(surface) === PDF
+        restrict_pdf_version!(surface, config.pdf_version)
```

Comment:
This should be optional, so that output remains unchanged.

Reply:
You mean to set `pdf_version` to `nothing` (or similar) by default and skip restricting the PDF version in that case?

Reply:
Yep, you elaborated more on this in the last comment but the Cairo version may change, or the user may use their own local Cairo which may not support that specific PDF version, so best to be safe

Reply:
The version can now be an empty string, in which case no restriction is applied.